src/Microsoft.PowerShell.Commands.Utility/commands/utility/select-object.cs

```diff
@@ -414,7 +419,7 @@ private void ProcessParameter(MshParameter p, PSObject inputObject, List<PSNoteP
                 }
             }
 
-            if (expressionResults.Count == 0)
+            if (expressionResults.Count == 0 && !ex.HasWildCardCharacters)
             {
                 //Commented out for bug 1107600
```

Comment:
Could you please remove the commented-out code snippet here? It's useless and there is no way to figure out what `bug 1107600` is now :)